pyproject.toml

```diff
@@ -58,15 +60,8 @@ docs = [
     "sphinx-toolbox==3.5.0",
     "sphinx-lint==0.8.1",
 ]
-# Python-apt bindings for specific Ubuntu versions.
-# Up to date package links can be found at https://launchpad.net/ubuntu/+source/python-apt
-# Note: These extras can break requirements from other packages, so
-# do not use them in dependencies unless you know what you're doing.
-jammy-dev = [
-    "python-apt@https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu2/python-apt_2.4.0ubuntu2.tar.xz;sys_platform=='linux'"
-]
-lunar-dev = [
-    "python-apt@https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.5.3ubuntu1/python-apt_2.5.3ubuntu1.tar.xz;sys_platform=='linux'"
+apt = [
```

Comment:
I guess the build failure is because this needs to be used wherever jammy-dev and lunar-dev were?